packages/rum-core/test/common/context.spec.js

```diff
@@ -211,7 +214,12 @@ describe('Context', () => {
       },
       ...userContext
     })
-
+    expect(pageloadTr.context.page.netinfo).toEqualOrUndefined({
```

Comment:
IMO, using `toEqualOrUndefined` is not a good idea, instead we should detect if the feature that is required for this to work exist and we should add explicit checks for the outcomes. With this matcher we don't know whether the result was undefined because of a bug or because the api is not supported.

Reply:
If i get it right, do you want to test it only for supported browsers? Like we do explicit check like 

```js
if (connection) {
  expect(measuredInfo).toBe(actualInfo)? 
}
```

I was trying to express this using the matcher, May be i missed something. We can also do explicit checks in tests if its confusing. 

````

Reply:
correct! also check the else condition, i.e:
if (connection) {
  expect(measuredInfo).toBe(actualInfo)
} else {
 expect(measuredInfo).toBe(undefined)
}

Reply:
Yeah sounds good 👍